### ISSUE TITLE
resume not work

### ISSUE THREAD

**Issue by alicera** [ultralytics/ultralytics#878]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

```
yolo detect train data=custom.yaml model=yolov8m.pt epochs=100 imgsz=640 batch=24 device=0,1,2,3 min_memory=True resume=runs/detect/train/weight/last.pt
```

 data=custom.yaml , not coco.yaml
The content of args.yaml is custom.yaml
```
Dataset 'coco.yaml' not found ⚠️, missing paths ['/work/AI/YOLOv8/datasets/coco/val2017.txt']
Downloading https://github.com/ultralytics/yolov5/releases/download/v1.0/coco2017labels-segments.zip to /work/AI/YOLOv8/datasets/coco2017labels-segments.zip...
 43%|█████████████████████████████████████████████████████████▍                                                                           | 72.7M/169M [00:10<00:29, 3.45MB/s] 44%|█████████████████████████████████████████████████████████▉                                                                           | 73.5M/169M [00:10<00:13, 7.40MB/s]
T
```

**Comment by bit-scientist** [comment]
@alicera  you don't need `model` part once you use `resume=path/to/best.pt`. Try it.

**Comment by alicera** [comment]
No, without model part, it will load the default yolov8n.pt

**Comment by bit-scientist** [comment]
Then try it as following: `yolo task=detect mode=train model=your_weights_path/last.pt device=0 resume=True`

**Comment by aistar1** [comment]
I found the two command work.
what is the different? (with or without resume arg)
because they are start from 27 epoch that break epoch
```
yolo detect train data=custom.yaml model=runs/detect/train/weight/last.pt device=0,1,2,3 min_memory=True resume=True
```
```
yolo detect train data=custom.yaml model=runs/detect/train/weight/last.pt device=0,1,2,3 min_memory=True
```

**Comment by bit-scientist** [comment]
the one with `resume` continues the epoch from where it stopped, whereas the second one starts the epoch from 1 just as you would normally train. Without `resume`, you're not using a pretrained yolov8n.pt, but using `runs/detect/train/weight/last.pt`.  So I think these two essentially do the same trainng. What's your say on that @Laughing-q ?

**Comment by Laughing-q** [comment]
@aistar1 @bit-scientist with `resume=True` it'll start resume training from where it stopped. without `resume=True` it'll treat your `last.pt` as a pretrained model and start training from scratch.
the correct usage of resume is:
```bash
yolo detect train model=last.pt resume=True
```
for python:
```python
model = YOLO("last.pt")
model.train(resume=True)
```

**Comment by dickymuhr** [comment]
> @aistar1 @bit-scientist with `resume=True` it'll start resume training from where it stopped. without `resume=True` it'll treat your `last.pt` as a pretrained model and start training from scratch. the correct usage of resume is:
> 
> ```shell
> yolo detect train model=last.pt resume=True
> ```
> 
> for python:
> 
> ```python
> model = YOLO("last.pt")
> model.train(resume=True)
> ```

when resume training, is the learning rate scheduler continue or it uses starting learning rate?

**Comment by Laughing-q** [comment]
@dickymuhr continue
